source/collections.txt

```diff
@@ -35,6 +35,13 @@ information for each collection in the selected database:
 
 - Collection name
 - Number of :doc:`documents </documents>` in the collection
+ 
+  .. note::
+
+    |compass-short| bases the document count that appears on the 
+    :guilabel:`Collections` screen on metadata. This count might differ 
```

Comment:
Same question as the other PR: Would specifying what meta data the doc count is based on or linking the a description of the metadata be helpful?